tests/cli/dorq/workflow/test_logs.py

```diff
@@ -63,67 +63,61 @@ def test_no_download_dir():
                 wf_run_resolver=wf_run_resolver,
             )
 
+            return action
+
+        @staticmethod
+        def test_no_download_dir(action):
+            # Given
+            # CLI inputs
+            wf_run_id = "<wf run ID sentinel>"
+            config = "<config sentinel>"
+            download_dir = None
+
             # When
             action.on_cmd_call(
                 wf_run_id=wf_run_id, config=config, download_dir=download_dir
             )
 
             # Then
             # We should pass input CLI args to config resolver.
-            presenter.show_error.assert_not_called()
-            config_resolver.resolve.assert_called_with(wf_run_id, config)
+            action._presenter.show_error.assert_not_called()
+            action._config_resolver.resolve.assert_called_with(wf_run_id, config)
 
             # We should pass resolved_config to run ID resolver.
-            wf_run_resolver.resolve_id.assert_called_with(wf_run_id, resolved_config)
+            resolved_config = action._config_resolver.resolve.return_value
+            action._wf_run_resolver.resolve.assert_called_with(
+                wf_run_id, resolved_config
+            )
 
             # We should pass resolved values to run repo.
-            wf_run_repo.get_wf_logs.assert_called_with(
-                wf_run_id=resolved_id, config_name=resolved_config
+            resolved_wf_run_id = action._wf_run_resolver.resolve.return_value
+            action._wf_run_repo.get_wf_logs.assert_called_with(
+                wf_run_id=resolved_wf_run_id,
+                config_name=resolved_config,
             )
 
             # We expect printing the workflow run returned from the repo.
-            presenter.show_logs.assert_called_with(fake_logs)
+            logs = action._wf_run_repo.get_wf_logs.return_value
+            action._presenter.show_logs.assert_called_with(logs)
 
             # We don't expect any dumps.
-            assert dumper.mock_calls == []
+            assert action._dumper.dump.mock_calls == []
 
         @staticmethod
-        def test_download_dir_passed():
+        def test_download_dir_passed(action):
             # Given
             # CLI inputs
             wf_run_id = "<wf run ID sentinel>"
             config = "<config sentinel>"
             download_dir = Path("/cool/path")
 
-            # Resolved values
-            resolved_id = "<resolved ID>"
-            resolved_config = "<resolved config>"
-
-            path_to_logs = "returns whatever"
-
-            # Mocks
-            presenter = create_autospec(WrappedCorqOutputPresenter)
-            dumper = create_autospec(LogsDumper)
-            dumper.dump.return_value = path_to_logs
+            # # Resolved values
+            # resolved_id = "<resolved ID>"
+            # resolved_config = "<resolved config>"
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏left over?